fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/RepositoryPaths.java

```diff
@@ -92,12 +126,19 @@ public static String getVocabulariesBase() {
         return vocabulariesBase;
     }
 
+    /**
+     * Initializes paths from repository base uri
+     *
+     * @param uri base uri for repository
+     */
     private static void setContentBase(String uri) {
         baseUri = uri;
         if (!baseUri.endsWith("/")) {
             baseUri += "/";
         }
-        contentBase = URIUtil.join(baseUri, RepositoryPathConstants.CONTENT_BASE);
-        contentRootPid = PIDs.get(URIUtil.join(contentBase, RepositoryPathConstants.CONTENT_ROOT_ID));
+        rootPid = new FedoraPID(REPOSITORY_ROOT_ID, REPOSITORY_ROOT_ID, null, URI.create(baseUri));
```

Comment:
why is REPOSITORY_ROOT_ID being passed in here twice?

Reply:
FedoraPIDs are usually constructed with an id and a qualifier. The id is the actual uuid or other identifier for the object. The qualifier is usually a root container in fedora (like "content" or "deposit") that groups together objects of the same purpose. In this branch I introduced PIDs for those root objects (see contentBasePid). None of those are in a subcontainer, so I introduced the REPOSITORY_ROOT_ID to use indicate that they are at the root rather than guessing if a blank string was intentional or not. It also adds rootPid, and I ended up deciding to give it REPOSITORY_ROOT_ID as its qualifier too, but it may be okay to just give it a blank for the qualifier, not sure.